administration/src/project-configs/getProjectConfig.ts

```diff
@@ -12,7 +12,11 @@ export interface PdfConfig {
   title: string
   templatePath: string | null
   issuer: string
-  infoToDetails: (info: CardInfo, region: Region, shorten: boolean) => string
+  elements?: {
+    staticQrCodes?: PdfQrCodeElementRenderer[]
+    dynamicQrCodes: PdfQrCodeElementRenderer[]
```

Comment:
Maybe make the names of the qr codes more specific? 

```suggestion
    staticVerificationQrCodes?: PdfQrCodeElementRenderer[]
    dynamicActivationQrCodes: PdfQrCodeElementRenderer[]
```